### DISCUSSION TITLE
Docker 部署 apollo 遇到 无法正常启动的问题

### DISCUSSION THREAD

**Discussion by JLQ5** [apolloconfig/apollo#4658] [Q&A]
按照官方使用文档 我拉取了最新的镜像，并且按照官方文档中的命令进行了部署
`docker run --platform linux/amd64 -p 8080:8080 \
    -e SPRING_DATASOURCE_URL="jdbc:mysql://<本机连接的wifi IP>:3306/ApolloConfigDB?characterEncoding=utf8" \
    -e SPRING_DActmp/logs:/opt/logs --name apollo-configservice apolloconfig/apollo-configservice:latest -n apollo
`
命令执行 后 容器正常running
<img width="1404" alt="截屏2022-11-25 15 41 11" src="https://user-images.githubusercontent.com/26503434/203927290-a3405c3f-51c8-4876-8b3c-2b8440c89ce5.png">

但是 进入容器后发现 启动进程卡死在了 java -version 里
<img width="1433" alt="截屏2022-11-25 15 44 32" src="https://user-images.githubusercontent.com/26503434/203928224-4c86479d-00d8-48f4-81f9-32f007499e5d.png">

我后面kill了启动进程 手动sh -x startup.sh 还是一样
<img width="1434" alt="截屏2022-11-25 15 55 12" src="https://user-images.githubusercontent.com/26503434/203929813-a646e8be-b0e3-458e-bb63-883de4de7d93.png">

**Comment by nobodyiam** [comment]
Are you running Apple M1? 
The [docker file](https://github.com/apolloconfig/apollo/blob/master/apollo-configservice/src/main/docker/Dockerfile#L32) is built againt amd64 instead of arm64, you may modify it and build the image on your own.

> FROM --platform=linux/arm64 openjdk:8-jre-slim

**Reply by JLQ5** [reply]
yes i‘m runnning Apple M1
I will modify docker file to check it
Thanks
